python/vowpalwabbit/pyvw.py

```diff
@@ -1157,6 +1157,47 @@ def __str__(self):
             ["{}:{}:{}".format(c.action, c.cost, c.probability) for c in self.costs]
         )
 
+class ccb_label(abstract_label):
+    """Class for conditional contextual bandits VW label"""
+
+    def __init__(self, type = "unset", cost=0, top_class=0, top_probability=0, explicitly_included_actions=[]):
+        abstract_label.__init__(self)
+        if isinstance(cost, example):
+            self.from_example(cost)
+        else:
+            self.type = type
```

Comment:
This object does not map to the contextual bandit label object. See [here](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Conditional-Contextual-Bandit#label-type) for what it contains.

I was thinking something along the lines of:
```Python
class SlotOutcome:
  cost: float
  action_probs: List[Tuple[int, float]]

class CCBLabel:
  type: CCBLabelType
  explicit_included_actions: List[int]
  weight: float
  outcome: Optional[SlotOutcome]
```

Reply:
@jackgerrits Updated it. Also, I think we should use snake case for naming classes?

Reply:
This file is far from consistent, but for new code we should try and follow the [PEP 8 guideline](https://www.python.org/dev/peps/pep-0008/#class-names) of PascalCase for classes.